tox.ini

```diff
@@ -1,12 +1,30 @@
 [tox]
-envlist =
-    py26,py27,py32,py33,py34
+envlist = flake8,
+          py27,
+          py35,
+          py36,
+          py37,
+          py38,
+          pypy,
+          pypy3,
+          coverage
 
 [testenv]
-deps =
-    zope.testing
-    zope.testrunner
-setenv =
-    RUNNING_UNDER_TOX=1
```

Comment:
Now what was _that_ about?

Reply:
Actually I think that hack was what used to make tox work, and might make Travis now work.

Reply:
I am going to fix this by paving over with meta/config.